examples/zustand-todo-list/package.json

```diff
@@ -2,19 +2,33 @@
   "description": "This example shows how to build a collaborative to-do list with Liveblocks, React and Zustand.",
   "license": "Apache-2.0",
   "private": true,
+  "dependencies": {
+    "@liveblocks/client": "^0.17.7",
+    "@liveblocks/zustand": "^0.17.7",
+    "@testing-library/jest-dom": "^5.16.4",
+    "@testing-library/react": "^13.3.0",
+    "@testing-library/user-event": "^13.5.0",
+    "@types/jest": "^27.5.2",
+    "@types/node": "^16.11.45",
+    "@types/react": "^18.0.15",
+    "@types/react-dom": "^18.0.6",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-scripts": "5.0.1",
+    "typescript": "^4.7.4",
+    "zustand": "^3.7.2"
+  },
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
     "eject": "react-scripts eject"
   },
-  "dependencies": {
-    "@liveblocks/client": "^0.17.7",
-    "@liveblocks/zustand": "^0.17.7",
-    "react": "^18.1.0",
-    "react-dom": "^18.1.0",
-    "react-scripts": "^5.0.1",
-    "zustand": "^3.7.1"
+  "eslintConfig": {
+    "extends": [
+      "react-app",
+      "react-app/jest"
+    ]
```

Comment:
We don't use ESLint in any examples yet and when we do it'll be better to do it in one go and figure out a shared config, for now we should remove any specific configs.